output/transmit_rps.sh

```diff
@@ -26,12 +26,15 @@ declare -A properties
 SCENARIODIR=$PWD
 RUNPROPERTIES=$SCENARIODIR/run.properties
 
-if [ -e "$RUNPROPERTIES" ] ; then
-    echo "Found run.properties file"
-    allMessage "transmit_rps.sh found the run.properties file."
-else
+# get ASGS functions
+SCRIPTDIR=`sed -n 's/[ ^]*$//;s/path.scriptdir\s*:\s*//p' $RUNPROPERTIES`
+source $SCRIPTDIR/properties.sh
+source $SCRIPTDIR/monitoring/logging.sh
+source $SCRIPTDIR/platforms.sh
+
+if [ ! -e "$RUNPROPERTIES" ] ; then
     allMessage "transmit_rps.sh did NOT find the run.properties file."
-    return 1
+    exit 1
```

Comment:
Looking into this, I am a little confused how this script is being called since I am seeing conflicting implication in the commits. (e.g., `exit` is concerning because if it's not called in a sub-shell, it will end the `asgs_main.sh` process).

Reply:
@jasonfleming can you provide some insight?

Reply:
this is called in the post processing hook, which I thought was a sub-shell process.

Reply:
I checked out the latest `transmit_rps-fix` to see how `transmit_rps.sh` was being executed ... this script is one element of the $POSTPROCESS bash array of post processing scripts, and they each get executed one at a time as separate commands. Based on my understanding of the bash process model, this script should execute as its own process, and the `exit` command should only kill the `transmit_rps.sh` process. 
The post processing as a whole (as well as other processes that occur after a forecast scenario has been submitted) is executed within a subshell, i.e., 
```
( 
   ... 
) &
```
so if the exit did kill more than just the `transmit_rps.sh` (which I'm thinking it won't) it could at most kill the forecast scenario subshell. Not sure how this was executed in previous commits, but this is what I see in the latest on this branch.

Reply:
FWIW, that `&` causes an asynchronous child process; should we get rid of it (will still be a sub shell) or put a `wait` after it?

Reply:
I know that's out of scope for this, but just something to note.

Reply:
Hey @wwlwpd this is meant to be asynchronous, since the forecast scenarios are "fire and forget" ... the ASGS submits them and then leaves them with the job monitoring, post processing, and archiving code they are supposed to run, and just leaves them to it goes on to the next task (or next advisory cycle).

Reply:
I am going to merge this so we can have it for Henri